src/main/scala/is/hail/annotations/aggregators/RegionValueTakeByIntAggregator.scala

```diff
@@ -0,0 +1,50 @@
+package is.hail.annotations.aggregators
+
+import is.hail.annotations._
+import is.hail.expr.types.Type
+
+import scala.collection.mutable
+
+class RegionValueTakeByIntIntAggregator(n: Int, val inputType: Type, val keyType: Type) extends RegionValueAggregator {
```

Comment:
I think you can make a generic TakeBy and then have IntIntTakeBy inherit from that and just define a seqOp, calling into the generic seqOp((Any, Any)).